openlibrary/templates/type/list/view.html

```diff
@@ -41,7 +41,7 @@
     </div>
     <h1>$list.name</h1>
     <p class="collapse sansserif">
-        <span class="darkgreen"><strong>$:render_seed_count(len(list.seeds))</strong></span>
+        <span class="darkgreen" id="item_count"><strong>$:render_seed_count(len(list.seeds))</strong></span>
```

Comment:
IDs are a little brittle because they're global, so choosing longer names is better to avoid collisions. Also we (generally) prefer hyphens to underscore in HTML classes/IDs.

```suggestion
        <span class="darkgreen" id="list-items-count"><strong>$:render_seed_count(len(list.seeds))</strong></span>
```